sys/posix/pthread/include/pthread_cond.h

```diff
@@ -24,6 +24,10 @@
 #   include "msp430_types.h"
 #endif
 
+#if defined(__APPLE__)
```

Comment:
If you see a reason why `__APPLE__` is better than `__MACH__` in some way, I'd suggest you open a fresh PR for changing the rest of RIOT.
Otherwise just check for `__MACH__` instead of `__APPLE__` here for consistency.